Django <3.1 requires psycopg2 ~2.8.0, as >=2.9 is incompatible

**Describe the bug**

Django 1.11.x fails to connect to PostgreSQL databases when using psycopg2 2.9 onwards, due to a failing timezone-related assertion.

See upstream for discussion:

* https://stackoverflow.com/questions/68024060/assertionerror-database-connection-isnt-set-to-utc
* https://github.com/psycopg/psycopg2/issues/1293#issuecomment-862835147


**Janeway version**

This is an issue for both the latest release (1.3.9) and master (d06be70), which don't pin the `psycopg2-binary` dependency.

**To Reproduce**

1. Create a fresh installation
2. Attempt to perform a database migration

Result:

```
[…]
  File "/usr/local/lib/python3.8/site-packages/django/db/utils.py", line 101, in inner
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/django/db/backends/postgresql/utils.py", line 6, in utc_tzinfo_factory
    raise AssertionError("database connection isn't set to UTC")
AssertionError: database connection isn't set to UTC
```

This can be resolved by pinning psycopg2-binary to less than 2.9, until Janeway uses a version of Django from 3.1 onwards.

--- a/.lando.yml
+++ b/.lando.yml
@@ -14,7 +14,7 @@ services:
       - grep '^deb ' /etc/apt/sources.list | perl -pe 's/deb /deb-src /' >> /etc/apt/sources.list
       - apt-get update -qq && apt-get install -y python3-lxml pylint libxml2-dev libxslt1-dev python3-dev zlib1g-dev lib32z1-dev libffi-dev libssl-dev libjpeg-dev && apt-get build-dep -y lxml
     build:
-      - cd /app && pip install --upgrade pip && pip install Cython && pip install psycopg2 && pip install -r requirements.txt &&  pip install -r dev-requirements.txt
+      - cd /app && pip install --upgrade pip && pip install Cython && pip install psycopg2-binary~=2.8.0 && pip install -r requirements.txt &&  pip install -r dev-requirements.txt
     scanner: true
     overrides:
       ports:

--- a/requirements.txt
+++ b/requirements.txt
@@ -31,7 +31,7 @@ mock
 mysqlclient==1.3.12
 pdfkit
 Pillow==7.1.0
-psycopg2-binary
+psycopg2-binary~=2.8.0
 pyasn1==0.1.9
 pycparser==2.14
 PyJWT==1.6.1

